Generators/include/Generators/Generator.h

```diff
@@ -17,6 +17,11 @@
 #include "FairGenerator.h"
 #include "TParticle.h"
 #include "Generators/Trigger.h"
+#include "CCDB/BasicCCDBManager.h"
```

Comment:
only include headers in headers if it cannot be avoided. Better to include in source file.